lib/slack/src/index.ts

```diff
@@ -5,13 +5,75 @@ const app = express()
 
 const port = process.env.PORT || 3001
 
+app.use(express.urlencoded({ extended: true }))
+
+app.get('/hello', async (_req, res) => {
+  res.send("world")
+})
+
 app.get('/ping', async (_req, res) => {
   const response = await fetch(`http://localhost:${daprPort}/v1.0/invoke/hr.hr/method/ping`)
   const body = await response.text()
   res.send("Respose from hr api: " + body)
 
 })
 
+// This is expecting a request from slack, with application/x-www-form-urlencoded data like below.
+// Details at https://api.slack.com/interactivity/slash-commands#app_command_handling
+// This (encoded) data is also saved as exampleRequest.txt for local testing with curl
+// {
+//   token: 'YOlSzbHSkwCPwdqZ3eHbHW1U',
+//   team_id: 'T02A80YKK',
+//   team_domain: 'redbadger',
+//   channel_id: 'D01R6L9NQ3S',
+//   channel_name: 'directmessage',
+//   user_id: 'U01QP0VJWJG',
+//   user_name: 'cedd.burge',
+//   command: '/get_manager',
+//   text: '@cedd.burge',
+//   api_app_id: 'A01SHH2QF8S',
+//   is_enterprise_install: 'false',
+//   response_url: 'https://hooks.slack.com/commands/T02A80YKK/2001166095842/CZprfzCQr6UoWSc5d0DiZivb',
```

Comment:
Fingers crossed there's nothing secret in here (or you scrambled it sufficiently).

Reply:
Hi David, this is a good point, and these details are repeated in exampleRequest.txt.

I think there is nothing sensitive, but probably another pair of eyes would be useful.
 - The token is deprecated and we don't use it (and won't)
 - The response_url and similar are short lived, so are no longer valid
 - The api_app_id, team_id, channel_id and user_id are real things, but I don't think they are sensitive ([signed secrets](https://api.slack.com/authentication/verifying-requests-from-slack) are how we should verify that a message is genuinely coming from slack, when we get round to doing that)

What do you think? Potentially I can recreate some of these id's if needs be.

Reply:
The main thing I was worried about was response_url, but if it's short-lived then that's fine. If `token` is also not usable for anything then I think we're fine.